packages/ui-components/src/components/radio/radio.base.scss

```diff
@@ -15,11 +15,11 @@
 }
 
 .radio-container {
-	width: 100%;
 	display: flex;
 	align-items: center;
 	cursor: pointer;
 	user-select: none;
+	width: fit-content;
```

Comment:
https://caniuse.com/?search=fit-content
Can we try other approach for this? Some browsers don't have full support to fit-content. 😞